### PR TITLE
Mvnv improve

### LINKED ISSUE #97: Add description to MVNV training component
Like this: 

Image

### PR DESCRIPTION
**Pull request by dyr429**
### Does this PR close any open issues?
Closes #97 #98

### Give a longer description of what this PR addresses and why it's needed
Solved style issues in MVNV study:

- Remove the white gap at the top
- Remove the "help" menu 
- Move the search to the top left
- Make the move the clear button next to it
- Move the legend to the top right
- updated training page

### Provide pictures/videos of the behavior before and after these changes (optional)
image
image
image

### Are there any additional TODOs before this PR is ready to go?
TODOs:
- [ 1] Add help to Revisit help modal. Make it support images.

### CODE REVIEW

**Comment by alexsb** [comment]
Looks, great, though I'd make this even skinnier: 

image


If you have two legends, put them next to each other. And you can make the square smaller and label to the left/right instead of top/bottom.

**Comment by alexsb** [comment]
Also, the search box probably doesn't need to be that wide, and neither does the clear button.

**Comment by dyr429** [comment]
That makes sense, I can compress it further.

File: public/js/training/mvnv/mvnv-training.html
Comment (hilsonshrestha) [inline public/js/training/mvnv/mvnv-training.html:1]: Should this file be deleted?

File: public/js/training/mvnv/audioAdjMatrix.mp4
[the remark this answers is not in the document]
Comment (dyr429) [reply public/js/training/mvnv/audioAdjMatrix.mp4:1]: Yeah, that make sense. will delete them

**Pull request merged**